source/patterns/@aws-solutions-constructs/core/lib/sqs-helper.ts

```diff
@@ -38,6 +40,19 @@ export interface BuildQueueProps {
      * @default - Default props are used.
      */
     readonly deadLetterQueue?: sqs.DeadLetterQueue
+    /**
+     * Use a KMS Key, either managed by this CDK app, or imported. If importing an encryption key, it must be specified in
+     * the encryptionKey property for this construct.
+     *
+     * @default - false (encryption enabled with a KMS key managed by SQS).
+     */
+    readonly enableEncryption?: boolean
```

Comment:
Can this parameter be renamed to indicate its required only for Customer Managed KMS Key ? e.g. `enableEncryptionWithCustomerManagedKey`